pkg/sdn/plugin/networkpolicy.go

```diff
@@ -178,6 +182,10 @@ func (np *networkPolicyPlugin) GetNamespaces(vnid uint32) []string {
 	return np.vnids.GetNamespaces(vnid)
 }
 
+func (np *networkPolicyPlugin) GetMCEnabled(vnid uint32) bool {
```

Comment:
Is there a reason for saying MC rather than spelling out Muticast?  I'm not sure I'll remember MC == multicast in a couple of months...